pom.xml

```diff
@@ -25,7 +25,13 @@
       <version>5.9.2</version>
       <scope>test</scope>
     </dependency>
-    
+      <dependency>
```

Comment:
remove this dependency, only use junit5

Reply:
please, tell me ways lines I have to delete?Thank you

Reply:
Remove all lines from 28 - 33.

Reply:
done